2996_reflection/reflection.md

```diff
@@ -2437,16 +2439,29 @@ static_assert(dealias(^Y) == ^int);
 ```
 :::
 
-### `value_of`
+### `object_of`, `value_of`
 
 ::: std
 ```c++
 namespace std::meta {
+  consteval auto object_of(info r) -> info;
   consteval auto value_of(info r) -> info;
 }
 ```
 :::
 
+If `r` is a reflection of a variable _usable in constant expressions_, then `object_of(r)` is a reflection of the object designated by the variable. For all other inputs, `object_of(r)` is not a constant expression.
+
+::: std
+```c++
+int x;
+int &y = x;
+
+static_assert(^x != ^y);
+static_assert(object_of(^x) == object_of(^y));
```

Comment:
So I think this description isn't quite right because `x` isn't "usable in constant expressions", but `y` is?

But I do think we want these two assertions to work, yeah.

Reply:
Good call - What I want is "variable designating an object having static storage duration".